serverless.yml

```diff
@@ -67,6 +74,8 @@ functions:
           path: addhack
           method: post
           cors: true
+          authorizer: authorizerFunc
+
   joinhack:
```

Comment:
should this one be locked down as well?

Reply:
Yes that' valid, if that's not guarded then we will get empty userId.